packages/web/pages/guild/[guildname].tsx

```diff
@@ -81,7 +81,7 @@ export const getStaticProps = async (
 
   return {
     props: {
-      guild,
+      guild: guild || null,
```

Comment:
This will give the incorrect output when `guid` is `false` (or anything other than undefined).